recipes-bsp/grub/grub-efi/grub.cfg

```diff
@@ -0,0 +1,13 @@
+serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
+default=boot
+timeout=5
+
+menuentry 'boot'{
+linux /bzImage rootwait root=LABEL=root rootfstype=ext4 console=ttyS0,115200 earlyprintk=serial,ttyS0,115200
+}
+
+menuentry 'skl-boot'{
```

Comment:
@krystian-hebel should we somehow detect Intel/AMD and select proper menu entry?

Reply:
Since https://github.com/TrenchBoot/grub/commit/b1083c1077b64222f74612b8c32b66f0b2b66c9a we should be able to put both SKL and ACM in separate `slaunch_module` lines, and code should choose the appropriate one, or use the ACM provided by firmware (if present and "better" than user-provided one). I'm not sure if we've tested it, though. At least the best ACM from multiple passed ones is properly chosen.

Reply:
Dawid inadvertently tested it by following too much of blog post. It works, but after printing an error and stopping for a few seconds. I think this can be fixed by not returning `GRUB_ERR_BAD_FILE_TYPE` in such cases, but this needs testing.